src/client/scss/app.scss

```diff
@@ -31,6 +31,42 @@
     cursor: ns-resize;
 }
 
+.zoom-buttons-wrapper {
+    position: absolute;
+    top: 74px;
+    right: 30px;
+    display: flex;
+}
+
+.zoom-buttons-wrapper button {
+    border: none;
+    background-color: #6c757d;
+    box-shadow: 0 1px 4px rgba(0,0,0,.35);
+    font-weight: bold;
+    font-size: 23px;
+    color: #fff;
+    width: 42px;
+    height: 42px;
+    transition: all .3s;
+}
+
+.zoom-buttons-wrapper button:first-child {
```

Comment:
These four can probably be avoided if Bootstrap styles are used correctly.

Reply:
Fixed it in [this commit](https://github.com/tilk/digitaljs_online/pull/34/commits/624659632023675806103264a07c45a1526378c2).